### ISSUE TITLE
Error when getting currency from a quest

### ISSUE THREAD

**Issue by kballard** [SavedInstances/SavedInstances#74]
Every time I turn in a quest that rewards some form of currency, SavedInstances throws an error.

```
1x SavedInstances\SavedInstances-7.0.8.lua:1811: attempt to index local 'texture' (a number value)
SavedInstances\SavedInstances-7.0.8.lua:1811: in function `QuestIsDarkmoonMonthly'
SavedInstances\SavedInstances-7.0.8.lua:1836: in function <SavedInstances\SavedInstances.lua:1830>
[C]: in function `GetQuestReward'
FrameXML\QuestFrame.lua:161: in function <FrameXML\QuestFrame.lua:147>

Locals:
self = <table> {
 GetCurrentMapAreaID = <function> defined @SavedInstances\SavedInstances.lua:1818
 GetServerOffset = <function> defined @SavedInstances\SavedInstances.lua:768
 oi_cache = <table> {
 }
 questdebug = <function> defined @SavedInstances\SavedInstances.lua:396
 QuestInfo = <function> defined @SavedInstances\SavedInstances.lua:328
 pvpdesertids = <table> {
 }
 icon = <table> {
 }
 histLiveCount = 0
 FindInstance = <function> defined @SavedInstances\SavedInstances.lua:976
 UpdateToonData = <function> defined @SavedInstances\SavedInstances.lua:1568
 GetRegion = <function> defined @SavedInstances\SavedInstances.lua:784
 InGroup = <function> defined @SavedInstances\SavedInstances.lua:2825
 chatMsg = <function> defined @SavedInstances\SavedInstances.lua:338
 CategorySize = <function> defined @SavedInstances\SavedInstances.lua:1063
 formatNumber = <function> defined @SavedInstances\SavedInstances.lua:454
 GetNextDailyResetTime = <function> defined @SavedInstances\SavedInstances.lua:809
 OrderedCategories = <function> defined @SavedInstances\SavedInstances.lua:1256
 instanceException = <function> defined @SavedInstances\SavedInstances.lua:1161
 db = <table> {
 }
 diff_strings = <table> {
 }
 LFRInstances = <table> {
 }
 transInstance = <table> {
 }
 IsDetached = <function> defined @SavedInstances\SavedInstances.lua:3358
 config = <table> {
 }
 QuestIsDarkmoonMonthly = <function> defined @SavedInstances\SavedInstances.lua:1804
 SkinFrame = <function> defined @SavedInstances\SavedInstances.lua:682
 Categories = <table> {
 }
 histTextUpdate = <function> defined @SavedInstances\SavedInstances.lua:3029
 HistoryEvent = <function> defined @SavedInstances\SavedInstances.lua:2860
 SetupVersion = <function> defined @SavedInstances\SavedInstances.lua:2601
 InstanceCategory = <function> defined @SavedInstances\SavedInstances.lua:1043
 histReapTime = 3600
 resetDetect = SavedInstancesResetDetectHiddenFrame {
 }
 HideDetached = <function> defined @SavedInstances\SavedInstances.lua:3361
 playedpending = false
 UpdateInstanceData = <function> defined @SavedInstances\SavedInstances.lua:1355
 seencds = <table> {
 }
 GetNextWeeklyResetTime = <function> defined @SavedInstances\SavedInstances.lua:868
 WorldBosses = <table> {
 }
 revision = 550
 skillwarned = <table> {
 }
 region = "US"
 histOldest = "n/a"
 debug = <function> defined @SavedInstances\SavedInstances.lua:342
 resetDays = <table> {
 }
 core = <table> {
 }
 activeHolidays = <table> {
 }
 LDB = <table> {
 }
 playedreg = <table> {
 }
 scaleCache = <table> {
 }
 UpdateInstance = <function> defined @SavedInstances\SavedInstances.lua:1471
 delayUpdate = 1511337685.2856
 ShowDetached = <function> defined @SavedInstances\SavedInstances.lua:3372
 oc_cache = <table> {
 }
 LookupInstance = <function> defined @SavedInstances\SavedInstances.lua:1012
 GetSeasonCurrency = <function> defined @SavedInstances\SavedInstances.lua:2370
 version = "7.0.8"
 idtext = <function> defined @SavedInstances\config.lua:45
 Indicators = <table> {
 }
 UpdateCurrency = <function> defined @SavedInstances\SavedInstances.lua:1754
 OrderedInstances = <function> defined @SavedInstances\SavedInstances.lua:1243
 updateSpellTip = <function> defined @SavedInstances\SavedInstances.lua:1549
 currency = <table> {
 }
 normalizeName = <function> defined @SavedInstances\SavedInstances.lua:950
 BonusRollShow = <function> defined @SavedInstances\SavedInstances.lua:4749
 instancesUpdated = true
 L = <table> {
 }
 specialQuests = <function> de
```

**Comment by Trid3nt1** [comment]
You're using 7.0.8 (A year old) - Have you tried grabbing the updated version 7.0.12? It can all be updated via the Curse / Twitch Client now too fyi.

I've personally not had this error come up, so I'm thinking it's something that's been fixed in between versions already.

**Comment by kballard** [comment]
You said a year old, but at the time that I filed this ticket, it was the latest available version on Curse. You didn't release v7.0.11 until 6 days ago (and v7.0.12 yesterday). I'll definitely try the new versions, but your response was pretty strange as it implied that I was not using the latest version.

**Comment by Trid3nt1** [comment]
Sorry, I didn't mean for it to come across like that. I was simply pointing out the age of your version, as in there'd been 7 Alphas 2 Releases between your version and live, in that time I am sure the issue you're having has been resolved.

I understand your ticket is ~16 days old, which is why I pointed out the new versions as they've not long been changed to "Release". Alot of people still manually update addons, don't run the Curse Client every day/week etc.. So it was just to let you know.
